IO-pc-local/src/app/lobby/lobby.service.ts

```diff
@@ -1,26 +1,43 @@
 import { Injectable } from '@angular/core';
-import { BehaviorSubject, Observable } from 'rxjs';
+import { Subject, Observable } from 'rxjs';
 import { Student } from './shared/model';
 
 @Injectable({
   providedIn: 'root'
 })
 export class LobbyService {
-  private readonly _students: BehaviorSubject<Student[]> = new BehaviorSubject([] as Student[]);
-  public students$: Observable<Student[]> = this._students.asObservable();
+  public readonly students = new Subject<Student[]>();
   private sse: EventSource | null = null;
 
+
   constructor() {
-    // this.sse = new EventSource("http://localhost:8000/", 
-    // { withCredentials: true, });
-    this._students.next(
-      [{name: "Maciuś"}, {name: "Zuzia"}, {name: "Franek"}, {name: "Janek"}]
-    )
+    this.sse = new EventSource("http://localhost:8000/game/lobby", 
```

Comment:
Super byłoby wyjąć to `http://localhost:8000` do jakiejś stałej w czymś typu `config.ts`